string drone_id
geometry_msgs/Point position
float32 battery_level
bool is_emergency